packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -182,11 +229,26 @@ exit 0
 %{_datadir}/rhsm-plugins/fqdn.*
 
 %files tracer
-%{_sysconfdir}/yum/pluginconf.d/tracer_upload.conf
+%{plugins_dir}/tracer_upload.py*
+%{katello_libdir}/tracer.py*
+%{plugins_confdir}/tracer_upload.conf
+
+%if %{dnf_install}
+%{katello_libdir}/__pycache__/tracer.*
+%{plugins_dir}/__pycache__/tracer_upload.*
+%else%
```

Comment:
I'm surprised `%else%` works. Shouldn't it be `%else`?

Reply:
I'm surprised too, will fix